Defs/Ammo/Pistols/9mm Glisenti.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+	<!-- ========== Based off of 9x19mm Parabellum ========== -->
+
+	<ThingCategoryDef>
+		<defName>Ammo9mmGlisenti</defName>
+		<label>9mm Glisenti</label>
+		<parent>AmmoPistols</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberPistol</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>

```

Comment:
While the 9mm variant is novel, I'd consider just letting the pistol use 9mm Para and call it a day. While not completely historically accurate, the dimensions between the two are identical and the matter of the powder charge is kind of splitting hairs.